### ISSUE TITLE
self.write_order_log missing from base.py, breaks RO

### ISSUE THREAD

**Issue by thehapax** [Codaone/DEXBot#593]
Method appears to be missing in latest devel branch merge from base.py
```   
 def write_order_log(self, worker_name, order):
        """ Write order log to csv file
            :param string | worker_name: Name of the worker
            :param object | order: Order that was fulfilled
        """
        operation_type = 'TRADE'

        if order['base']['symbol'] == self.market['base']['symbol']:
            base_symbol = order['base']['symbol']
            base_amount = -order['base']['amount']
            quote_symbol = order['quote']['symbol']
            quote_amount = order['quote']['amount']
        else:
            base_symbol = order['quote']['symbol']
            base_amount = order['quote']['amount']
            quote_symbol = order['base']['symbol']
            quote_amount = -order['base']['amount']

        message = '{};{};{};{};{};{};{};{}'.format(
            worker_name,
            order['id'],
            operation_type,
            base_symbol,
            base_amount,
            quote_symbol,
            quote_amount,
            datetime.datetime.now().isoformat()
        )

        self.orders_log.info(message)
```

## Steps to Reproduce the Problem
Affects Relative orders initialization, check_orders when not using custom expiration

@joelvai

**Comment by thehapax** [comment]
also from the init, missing:
```
       self.orders_log = logging.LoggerAdapter(
            logging.getLogger('dexbot.orders_log'), {}
        )
```

**Comment by bitfag** [comment]
I'd rather remove this order log because it's not accurate anyway. Reliable data can be obtained from account history / elasticsearch wrapper.

**Comment by thehapax** [comment]
@bitfag has #204 already been implementated? If not lets open a task for it. 

making a pull request to eliminate self.write_order_log from base and RO

**Comment by bitfag** [comment]
`write_order_log()` is current implementation of #204. A new implementation has not been done (I was assigned to another task), so only plugin infrastructure has been implemented (#500 and #502).
